.github/workflows/ci.yml

```diff
@@ -3,31 +3,51 @@ name: CI
 on: [push, pull_request]
 
 jobs:
-  ci:
+  nightly:
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
-        rust-toolchain: [nightly]
+        features: ["", "--all-features"]
```

Comment:
A better implementation:

```tml
matrix:
  suits:
    - toolchain: stable
      features: ""
    - toolchain: nightly
      features: ""
    - toolchain: nightly
      features: "--all-features"
  targets: [x86_64-unknown-linux-gnu, x86_64-unknown-none, riscv64gc-unknown-none-elf, aarch64-unknown-none-softfloat]
```

Then you can merge the `nightly` and `stable` jobs.